app/views/charge.njk

```diff
@@ -97,377 +97,383 @@
+              name: 'email-typo-sugestion',
+              errorMessage: {
+                text: __p("fieldErrors.fields.email.typo"),
+                classes: 'govuk-!-width-three-quarters'
               },
-              {
-                id: 'email-uncorrected',
-                value: email,
-                text: email
+              items: [
+                {
+                  id: 'email-corrected',
+                  value: typos.full,
+                  html: typos.address + '@<strong>' + typos.domain + '</strong>',
+                  checked: true
+                },
+                {
+                  id: 'email-uncorrected',
+                  value: email,
+                  text: email
+                }
+              ]
+            })
+          }}
+        {% endif %}
+        <div>
+        {% if typos %}
+          {{
+            govukButton({
+              text: __p("commonButtons.continueButton"),
+              attributes: {
+                id: "submit-card-details",
+                "data-click-events": "true",
+                "data-click-category":"Typo made",
+                "data-click-action": "Corrected email: " + typos.domain
               }
-            ]
-          })
-        }}
-      {% endif %}
-      <div>
-      {% if typos %}
-        {{
-          govukButton({
-            text: __p("commonButtons.continueButton"),
-            attributes: {
-              id: "submit-card-details",
-              "data-click-events": "true",
-              "data-click-category":"Typo made",
-              "data-click-action": "Corrected email: " + typos.domain
-            }
-          })
-        }}
-      {% else %}
-        {{
-          govukButton({
-            text: __p("commonButtons.continueButton"),
-            attributes: {
-              id: "submit-card-details"
-            }
-          })
-        }}
-      {% endif %}
-      </div>
-    </form>
-    <form id="cancel" name="cancel" method="POST" action="{{ post_cancel_action }}" class="form">
-      <div>
-        <input id="cancel-payment" type="submit" class="button-reset" value="{{ __p("commonButtons.cancelButton") }}" name="cancel">
-        <input id="csrf2" name="csrfToken" type="hidden" value="{{ csrf }}"/>
-      </div>
-    </form>
+            })
+          }}
+        {% else %}
+          {{
+            govukButton({
+              text: __p("commonButtons.continueButton"),
+              attributes: {
+                id: "submit-card-details"
+              }
+            })
+          }}
+        {% endif %}
+        </div>
+      </form>
+      <form id="cancel" name="cancel" method="POST" action="{{ post_cancel_action }}" class="form">
+        <div>
+          <input id="cancel-payment" type="submit" class="button-reset" value="{{ __p("commonButtons.cancelButton") }}" name="cancel">
+          <input id="csrf2" name="csrfToken" type="hidden" value="{{ csrf }}"/>
+        </div>
+      </form>
+    </div>
```

Comment:
I think if these indentation changes has been made in a separate commit, it would have been easier to review